release-notes/release-notes-2-11.md

```diff
@@ -147,7 +147,10 @@ in the [Data Connectors](https://github.com/odpi/egeria-data-connectors) GitHub
 ## Known Issues
 
 * It is recommended to use a chromium-based browser such as Google Chrome or Microsoft Edge, or Apple Safari for the Egeria React UI. Some parts of the UI experience such as Dino currently experience problems with Firefox. See [odpi/egeria-react-ui#96](https://github.com/odpi/egeria-react-ui/issues/96) .
-* When running the 'Understanding Platform Services' lab, ensure you run the 'egeria-service-config' notebook first and do not restart the python kernel before running this lab. See [#4842](https://github.com/odpi/egeria/issues/4842) .
+
+## Special note on Java Support
+
+* This is expected to be the last release supporting Java 8. The next release will commence the 3.x series of releases and require Java 11.
```

Comment:
I wonder do we want to talk about 3 being a new version to highlight the change

Reply:
I wonder if we need more detail that 'require Java 11'.
- I run with java 15 and that works. Maybe saying we now target compiles at java 11 bytecodes would be better
- maybe mention both at runtime and compile time.
-

Reply:
I do say it will commence the 3.x series of releases? I think that's sufficient?
On Java version, I've written more in the Java.md on release 3.0 which I hope is a little clearer (though we can refine that accordingly!) and where we have more space to elaborate. I wanted to add a simple reference here as a 'heads-up'. With that goal I think saying require Java 11 is sufficient EVEN THOUGH I agree technically - We require Java 11 as a min, and we build everything inc. maven artifacts/docker with Java 11. We also build with Java 16 which is kinda ok but we have one glitch with private symbols. In summary I'm inclued to just adopt the KISS principle in this statement

Reply:
That PR for master is at https://github.com/odpi/egeria/pull/5403